spec/controllers/questions/personal_info_controller_spec.rb

```diff
@@ -7,6 +7,28 @@
     allow(subject).to receive(:current_intake).and_return(intake)
   end
 
+  describe "#edit" do
+    context "when a client said they need help getting an ITIN during triage" do
+      let(:intake) { create :intake, triage: create(:triage, id_type: "need_itin_help") }
+
+      it "sets hide_ssn_field to be truthy" do
+        get :edit
+
+        expect(assigns(:hide_ssn_field)).to be_truthy
```

Comment:
hmm i actually changed it from show_ssn_field to hide_ssn_field because I believe that the `hide` naming more accurately describes that show is the default and the exceptional case is to hide it.